### PR TITLE
cloud test setup documentation

### PR DESCRIPTION
**Pull request by grafnu**

### CODE REVIEW

**Comment by grafnu** [comment]
For  now, I would just review this for basic english, but save a vetting walkthrough as part of the cleanup/test documentation pass.

File: docs/cloud_tests.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+A number of additional setup steps are required for enabling testing against "smart devices"
+that communicate with the cloud. The tests themselves are part of the `subset/cloud/test_udmi`
+module included in the standard DAQ distro. The same basic device-to-cloud validation test
+pipeline can be done manually and automatically (through DAQ); it's instructure to fully
+understand the manual test pipeline before engaging with the automated setup.
+
+## Manual Test Pipeline
+
+The overal device-to-cloud pipeline looks something like the following:
```
Comment (henry54809) [inline docs/cloud_tests.md:11]: typo

File: docs/cloud_tests.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+Setting environment variables from .travis.yml
+$ export DOCKER_STARTUP_TIMEOUT_MS=60000
+$ export DAQ_TEST=aux
+```
+
+### Other Travis Caveats
+
+Take note the URL in your browser's address bar when running Travis. You might be on either
+<code>travis-ci<b>.com</b></code> or <code>travis-ci<b>.org</b></code>. Any particular setup
+may end up across both sites for undertermined reasons. Please consult with your browser's
```
Comment (henry54809) [inline docs/cloud_tests.md:131]: typo

**Comment by grafnu** [comment]
Typos have been addressed, PTAL.

**Pull request merged**